server/controller/routes/admin/createEvent.js

```diff
@@ -13,34 +13,33 @@ const validateEvent = (req, res, next) => {
     date: yup.date().required(),
     duration: yup.number().required().positive().integer(),
   });
-
-  const result = schema.isValidSync(req.body);
-  if (!result) {
-    const err = new Error();
-    err.msg = 'invalid inputs';
-    err.status = 400;
-    return next(err);
+  try {
+    const result = schema.isValidSync(req.body);
```

Comment:
use `await` here with `isValid`
```suggestion
    const result = await schema.isValid(req.body);
```